index.html

```diff
@@ -1,1046 +1,1078 @@
+      confidentiality of CBOR data by encrypting it with symmetric or
+      asymmetric encryption algorithms. Our use case here, with
+      verifiable credentials is focused on signing, primarily through
+      use of COSE Sign1 operations.
+    </p>
+  </section>
 
+  <section>
+    <h2>Securing the Core VC Data Model</h2>
+    <p>
+      The use of Verifiable Credentials often involves the
+      representation and exchange of structured data in the form of
+      JSON-LD as this is the format of the core data model. While
+      JSON-LD provides a flexible and extensible format for describing
+      data, it is important to note that it also provides a linkage
+      between the data structure and semantic meaning of data.
+    </p>
+    <p>
+      This section outlines how JSON-LD encoded claimset can be secured
+      using either JOSE or COSE.
+    </p>
+    <p>
+      A benefit to this approach is that payloads can be made to conform
+      directly to the [[VC-DATA-MODEL]] without any mapping or
+      transformation.
+    </p>
     <section>
-      <h2>Securing JSON</h2>
-      <p>
-        This section
-        provides guidance on how to use JSON [[RFC7159]] claimsets with JWT registered claims to construct
-        a JWT that can be mapped to a verifiable credential.  This section also describes
-        how to use content types and token types to distinguish different representations of verifiable credentials.
-      </p>
-      <p class="note">
-        This representation relies on claims registered in the
-        <a href="https://www.iana.org/assignments/jwt/jwt.xhtml#claims">IANA JSON Web Token Claims Registry</a> whenever possible.
-      </p>
-      <p class="note">
-        Production of this representation does not use <code>vc+ld+json</code> as an input.
-      </p>
+      <h2>With JOSE</h2>
       <section>
-        <h2>Credential Header</h2>
-        <p><a data-cite="rfc7519#section-5.1">typ</a> MUST use the media type <code>vc+jwt</code>.</p>
-        <aside
-          class="example"
-          title="credential metadata (decoded JWT header)"
-        >
-          <pre class="json">
-{
-  "kid": "https://example.edu/issuers/14#key-0",
-  "alg": "ES256",
-  "typ": "vc+jwt"
-}
+        <h2>Securing JSON-LD Verifiable Credentials with JOSE</h2>
+        <p>
+          This section details how to secure data payloads with the type
+          <code>application/vc+ld+json</code>
+          with JOSE.
+        </p>
+        <p>[[rfc7515]] MAY be used to secure this media type.</p>
+        <p>
+          The <code>typ</code> parameter MUST be <code>vc+ld+jwt</code>
+        </p>
+        <p>
+          When present, the <code>cty</code> MUST be
+          <code>vc+ld+json</code>
+        </p>
+        <p>
+          See <a data-cite="rfc7515#section-4.1.10">Common JOSE Header
+          Parameters</a>
+          for additional details regarding usage of <code>typ</code> and
```

Comment:
`typ` only works for `JSON` payloads... same issue as https://github.com/w3c/vc-jwt/pull/112/files#r1245326420